src/passes/StackCheck.cpp

```diff
@@ -55,23 +48,12 @@ static void importStackOverflowHandler(Module& module, Name name) {
 static void addExportedFunction(Module& module, Function* function) {
   module.addFunction(function);
   auto export_ = new Export;
-  export_->name = export_->value = function->name;
+  export_->name = Names::getValidExportName(module, function->name);
```

Comment:
Will users of the generated code know to look for the new export name if there was a conflict?

Reply:
It seems like this can only happen by running the pass more than once, so it's probably someone running it manually anyhow. But, fair point that erroring on the export might be safer.